# This message contains information provided by introspective perception
# on the likelihood of failure, complemented with information about
# the corresponding navigation edge provided by the navigation graph planner
uint64 s0_id
uint64 s1_id
float64 failure_likelihood
uint8 failure_type
